quesma/quesma/search.go

```diff
@@ -254,9 +254,9 @@ func (q *QueryRunner) executePlan(ctx context.Context, plan *model.ExecutionPlan
 		return responseBody, err
 	}
 
-	if resp, err := q.checkProperties(ctx, plan, table, queryTranslator); err != nil {
-		return resp, err
-	}
+	//if resp, err := q.checkProperties(ctx, plan, table, queryTranslator); err != nil {
```

Comment:
`checkProperties` has a comment that it's deprecated and should be removed. This had false positives in newly generated queries (`SELECT * FROM (... UNION ALL ...)`) since it tried to find columns from that subqueries in schema and failed (because they were aliased slightly differently).